lib/lib_rover/include/actuators/motor_drivers/talon_srx.hpp

```diff
@@ -1,20 +1,19 @@
 #ifndef __TALONSRX_DRIVE_HPP__
 #define __TALONSRX_DRIVE_HPP__
 
-#if !defined(ESP32)
+#if !defined(ARDUINO_ESP32S2_DEV) && !defined(ARDUINO_ESP32S3_DEV)
```

Comment:
Why not ESP32

Reply:
see lines 25 @ 29 and 60 @ 65. For ESP32-S2 there are 20 bits timers but on esp32 there is only 14 bits timers. This will make sure if someone try with another chip (S, C2, C3, etc.) he'll need to check which timers are available for this specific esp32 chip and make the correct implementation. It should always be like this (always only allow the board for which it was developped). But we don't really have time right now. (It's a good issue to had to the repo).